### ISSUE TITLE
Downloaded file is not a full 360°

### ISSUE THREAD

**Issue by maciejopalinski** [fdd4s/streetview-dl#1]
How to reproduce:
1. Run (as per [README.md](https://github.com/fdd4s/streetview-dl/blob/c5b438dcc4359065dca3b72424701d4e99e70452/README.md)):
```sh
php ./streeview-dl.php 'https://www.google.com/maps/@41.3881837,2.1698939,3a,75y,143.45h,92.72t/data=!3m6!1e1!3m4!1sr3vUp9U2ss5fwoq1Roxizw!2e0!7i16384!8i8192' 
```

The file that gets downloaded is not a full version. There are parts missing on the left and right.

**Comment by maciejopalinski** [comment]
Another example:
```sh
php ./streeview-dl.php 'p-EPpErUWv2pjDvUC3N6rQ'
```

The building on the right gets cut a bit and entire building with 'Wegmans' on it is no longer there.
![sts-p-EPpErUWv2pjDvUC3N6rQ](https://github.com/fdd4s/streetview-dl/assets/44976946/810cbbff-5c7c-49fa-84dd-67bd22d5825e)
[Look on Google Maps](https://www.google.com/maps/@40.6982454,-73.980301,3a,84.9y,36.94h,99.2t/data=!3m6!1e1!3m4!1sp-EPpErUWv2pjDvUC3N6rQ!2e0!7i16384!8i8192?entry=ttu)

**Comment by samstommy** [comment]
yes, the resulting panorama is still shorter.  Trying to play around the number of tiles and resolution still, nothing "corrupted tiles if changed", will there be a fix for this in sight?

**Comment by EmmanuelPil** [comment]
Struggling with the same issue; playing with the zoom factor but I can't stich them right.

**Comment by Chloezu** [comment]
Made a pr that seems to fix the issue, just incase anyone from here was looking for that fix still!

**Comment by Chloezu** [comment]
alright so i finally remembered all of this and figured since its been ages i should release my python rewrite of this
https://github.com/Chloezu/Google-Streetview-DL
it should work just fine for all varieties of streetview image, some fixes are a little janky and should be re-written, but in my testing every update to the 'format' they've used has worked just fine
